modApiServer/src/org/aion/api/server/http/HttpServer.java

```diff
@@ -169,8 +169,7 @@ private static JSONObject process(final IRpc.Method _method, final long _id, fin
 
         case eth_getWork:
             // Header without nonce and solution , pool needs add new nonce
-            return processResult(_id, api.getBestBlock().getHeader().getHeaderBytes(true));
-
+                return processResult(_id, toHexString(HashUtil.h256(api.getBestBlock().getHeader().getHeaderBytes(true))));
```

Comment:
need review from @aion-6368726973 or @aion-Ross if this will break miner compatibility.

Reply:
https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getwork

Reply:
At this time it should not break the reference miners/pool as they are relying on the "getblocktemplate" call.

However why are we returning a hash of the block header? If we want to update this to return Equihash work we would need eth_getWork to return the entire block header. Our result will be quite different from the referenced eth_getWork.